src/DoctrineWriter.php

```diff
@@ -262,47 +287,50 @@ protected function disableLogging()
      */
     protected function reEnableLogging()
     {
-        $config = $this->entityManager->getConnection()->getConfiguration();
+        //TODO: do we need to add support for MongoDB logging?
+        if (!($this->objectManager instanceof \Doctrine\ORM\EntityManager)) return;
```

Comment:
Why? If ODM is used, it will just do nothing. This is expected behaviour. Logger should be disabled/enabled only for EntityManager